test/first_party/CMakeLists.txt

```
@@ -63,7 +61,11 @@ if (GCC_BIN_RV64)
 endif()
 
 if (NOT CROSS_COMPILER_COMMAND_RV32 OR NOT CROSS_COMPILER_COMMAND_RV64)
-    message(FATAL_ERROR "No suitable cross-compiler found. We recommend downloading Clang from https://github.com/llvm/llvm-project/releases")
+    if (CLANG_BIN)
+        message(WARNING "Your Clang compiler does not support RISC-V (see '${CLANG_BIN} -print-targets'). Please download one from https://github.com/llvm/llvm-project/releases")
+    else()
+        message(FATAL_ERROR "No suitable cross-compiler found. We recommend downloading Clang from https://github.com/llvm/llvm-project/releases")
+    endif()
```

```suggestion
    if (CLANG_BIN)
        message(WARNING "Your Clang compiler does not support RISC-V (see '${CLANG_BIN} -print-targets'). Please download one from https://github.com/llvm/llvm-project/releases")
    endif()
    message(FATAL_ERROR "No suitable cross-compiler found. We recommend downloading Clang from https://github.com/llvm/llvm-project/releases")
```

I believe this should still be an unconditional FATAL_ERROR since we didn't find a suitable cross compiler.

---

Oops. Good catch. Fixed.